pkgs/development/python-modules/slpp/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "slpp";
+  version = "1.2.3";
+
+  src = fetchPypi {
+    pname = "SLPP";
+    inherit version;
+    hash = "sha256-If3ZMoNICQxxpdMnc+juaKq4rX7MMi9eDMAQEUy1Scg=";
+  };
+
+  propagatedBuildInputs = [
+    six
+  ];
+
+  pythonImportsCheck = [
+    "slpp"
+  ];
+
```

Comment:
Could you please enable tests with pytestCheckHook or unittestCheckHook?

Reply:
They don't have any tests packaged in the PyPI archive, though.